### PR TITLE
Use system config dir instead of ~/.edgedb

### PR DESCRIPTION
**Pull request by jaclarke**
Copies changes from https://github.com/edgedb/edgedb-python/pull/197

### CODE REVIEW

File: src/platform.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    return path.join(xdgConfigDir, "edgedb");
+  };
+}
+
+export function getConfigDir(): string {
+  let configDir = _configDir();
+  if (!fs.existsSync(configDir)) {
+    configDir = path.join(homeDir(), ".edgedb");
+  }
```
Comment (fantix) [inline src/platform.ts:33]: I'm sorry about the delay - but there's an update in the Python binding to address comments: https://github.com/edgedb/edgedb-python/compare/aeb84cd...a1c817d?expand=1#diff-7390f121f7b0fa5c6846ff46ad4adfd547e45c0bef1dbf194a46873402a779cc

Comment (jaclarke) [reply src/platform.ts]: Ok, updated

**Review by fantix — APPROVED** [review]
Thanks!

**Comment by elprans** [comment]
@jaclarke, merge this?

**Pull request merged**